src/VirtoCommerce.ReturnModule.Web/Scripts/blades/return-list.tpl.html

```diff
@@ -0,0 +1,44 @@
+<div class="blade-static">
+    <div class="form-group">
+        <div style="display: flex;align-items: center;">
+            <div class="form-input" style="flex-grow: 1;">
+                <div class="form-input __other">
+                    <input placeholder="{{ 'platform.placeholders.search-keyword' | translate }}" ng-model="blade.searchKeyword" ng-keyup="$event.which === 13 && blade.refresh()" style="width: 80%;" />
+                    <button class="btn __other" type="button" style="position: relative;right: 45px;">
+                        <i class="btn-ico fa fa-remove" title="Clear" ng-click="clearKeyword()"></i>
+                    </button>
+                </div>
+                <p class="form-count">{{'platform.list.count' | translate}}: <span class="count">{{pageSettings.totalItems | number:0}}</span></p>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="blade-static __bottom" ng-if="pageSettings.itemsPerPageCount < pageSettings.totalItems" ng-include="'pagerTemplate.html'"></div>
+<div class="blade-content __xxlarge-wide">
+    <div class="blade-inner">
+        <div class="inner-block" ng-init="setGridOptions('returnlist-grid', {
+                    useExternalSorting: true,
+                    data: 'listEntries',
+                    rowTemplate: 'return-list.row.html',
+                    rowHeight: 61,
+                    columnDefs: [
+                                { name: 'returnNumber', displayName: 'return.blades.return-list.labels.return-number', sort: { direction: uiGridConstants.ASC } },
```

Comment:
You can sort only by one of columns. All other is redundand.

You may also move this code to js: https://github.com/VirtoCommerce/vc-module-order/blob/e12f3ab62d4885ce3df309d45454357bd88ef6fd/src/VirtoCommerce.OrdersModule.Web/Scripts/blades/customerOrder-list.js#L10